src/BizHawk.Emulation.Cores/Consoles/Nintendo/SNES/LibsnesApi.cs

```diff
@@ -34,7 +34,7 @@ static LibsnesApi()
 			}
 		}
 
-		private PeRunner _exe;
+		private Waterbox.Waterbox _exe;

```

Comment:
Uggg, waterbox is now `Waterbox.Waterbox`?  I forgot how awful naming a namespace and class the same thing was.

Reply:
Renamed